src/config/env.ts

```diff
@@ -83,7 +83,7 @@ export interface EnvVars {
 	APP_ID: string;
 	APP_URL: string;
 	APP_KEY: string;
-	WEBHOOK_SECRET: string;
+	WEBHOOK_SECRETS: string;
```

Comment:
/nit

I think you can put `JSON.parse(envVars.WEBHOOK_SECRETS);` inside the `const transforms: Transforms<EnvVars> = {` above so that the consumer doesn't need to `JSON.parse` everytime it is called. Especially webhook will be called a lots of time.

Reply:
updated